dom.bs

```diff
@@ -6758,10 +6790,20 @@ method steps are:
  and null otherwise.
  <!-- This is step 2 of "get an attribute by name", modified as appropriate -->
 
- <li><p>If <var>attribute</var> is null, create an <a>attribute</a> whose
- <a for=Attr>local name</a> is <var>qualifiedName</var>, <a for=Attr>value</a> is
- <var>value</var>, and <a for=Node>node document</a> is <a>this</a>'s <a for=Node>node document</a>,
- then <a lt="append an attribute">append</a> this <a>attribute</a> to <a>this</a>, and then return.
+ <li><p>If <var>attribute</var> is null, then:
```

Comment:
Wrapping is still wrong here. The `<li>` has multiple children so the `<p>` needs to be newlined and indented. The `<ol>` below doesn't seem indented enough either. Remember that it's a child of the `<li>`.